raml-headers-form.html

```diff
@@ -197,7 +223,7 @@
                     <label>Header name</label>
                     <input
                       is="anypoint-input"
-                      class="name-change-input"
+                      class="name-change-input anypoint-text-input"
```

Comment:
Maybe instead of adding special class for any input it is better to use anypoint input's mixins? That this change is about anyway? Anypoint inputs are styled as they are designed in our style guide.